### PR TITLE
Add new SSR1xxx-DC sku's that match their counterparts for port layout

### PR DESCRIPTION
**Pull request by agrawalkaushik**
WAN-2803 #time 10m

- [x] WAITING ON CLEI - DO NOT MERGE

### CODE REVIEW

**Review by BenMatase — APPROVED** [review]
Same clei and part num?

**Comment by agrawalkaushik** [comment]
> Same clei and part num?

@avijitvjuniper do you know if these new units are meant to have the same CLEI and part number?

**Comment by avijitvjuniper** [comment]
> > Same clei and part num?
> 
> @avijitvjuniper do you know if these new units are meant to have the same CLEI and part number?

Hi Kaushik, Ben, let me check if we can keep the same CLEI but the part number will change. This will feature a new 650-XXXXXX JPN

**Review by haberkornsam — APPROVED** [review]

File: devicemaps.json [hunk 1]
```diff
@@ -1041,6 +1041,14 @@
         ],
         "lte": []
       },
+      "SSR1300-DC": {
+        "clei": "INMJ410HRA",
+        "part_number": "650-180066",
```
Comment (avijitvjuniper) [inline devicemaps.json:1046]: Please correct the part number to 650-180064

File: devicemaps.json [hunk 2]
```diff
@@ -1218,6 +1226,14 @@
         ],
         "lte": []
       },
+      "SSR1400-DC": {
+        "clei": "INMJ510HRA",
+        "part_number": "650-180067",
```
Comment (avijitvjuniper) [inline devicemaps.json:1231]: Please correct the part number to 650-180065

File: devicemaps.json (hunk 1 above)
Comment (agrawalkaushik) [reply devicemaps.json:1046]: this is super confusing - as long as they put the right information in the EEPROM it doesn't matter to me. Thanks for catching it.

**Review by avijitvjuniper — COMMENTED** [review]
Looks good. Thank you!

**Pull request merged**